api/comments/serializers.py

```diff
@@ -48,12 +56,31 @@ class CommentSerializer(JSONAPISerializer):
     class Meta:
         type_ = 'comments'
 
+    def get_is_abuse(self, obj):
+        user = self.context['request'].user
+        if user.is_anonymous():
+            return False
+        return user._id in obj.reports
+
+    def get_can_edit(self, obj):
+        user = self.context['request'].user
+        if user.is_anonymous():
+            return False
+        return obj.user._id == user._id
+
+    def get_has_children(self, obj):
+        return bool(Comment.find(Q('target', 'eq', obj)).count())
```

Comment:
I would change this to`return Comment.find(Q('target', 'eq', obj)).count() > 0` just to be more explicit. Also, what does count() do in the case of an exception? Is the exception going to bubble and kill the whole response or will it incorrectly return false?